helper/testhelpers/ldap/ldaphelper.go

```diff
@@ -6,21 +6,15 @@ package ldap
 import (
 	"context"
 	"fmt"
-	"runtime"
-	"strings"
 	"testing"
 
-	hclog "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/cap/ldap"
+
 	"github.com/hashicorp/vault/sdk/helper/docker"
 	"github.com/hashicorp/vault/sdk/helper/ldaputil"
 )
 
 func PrepareTestContainer(t *testing.T, version string) (cleanup func(), cfg *ldaputil.ConfigEntry) {
-	// Skipping on ARM, as this image can't run on ARM architecture
-	if strings.Contains(runtime.GOARCH, "arm") {
```

Comment:
I was able to run these tests locally on my Mac M1, but if this causes test failures I'll revert these changes.